src/include/kdb.h

```diff
@@ -1525,6 +1512,13 @@ typedef struct _kdb_vftabl {
                                void *ad_info);
 
     /* End of minor version 0 for major version 8. */
+
+    krb5_error_code (*issue_pac)(krb5_context context, unsigned int flags,
+                                 krb5_const_principal client_princ,
+                                 krb5_db_entry *proxy,
+                                 krb5_timestamp authtime,
+                                 krb5_boolean is_cross_realm,
+                                 krb5_pac old_pac, krb5_pac new_pac);
```

Comment:
We will need to bump the major DB version for this change, and the new method belongs before the "End of minor version 0" comment for the new major version.

The new method will need a documentation comment, with particular attention to how the module should handle `flags`.

`is_cross_realm` is redundant with KRB5_KDB_FLAG_CROSS_REALM in `flags`, unless it's trying to convey something subtly different.

The fourth parameter should be named `client` instead of `proxy`.

Reply:
I'd been holding off on bumping the KDB version until this is closer to done, but I suppose that doesn't actually help anything.  Will make the change.

Reply:
I've added documentation, but I'm not sure what you have in mind for `flags`.  Probably what I have there is not what you meant :)